src/xbot/common/math/PID.java

```diff
@@ -79,6 +116,9 @@ public double calculate(double goal, double current,
             m_result = m_minimumOutput;
         }
         result = m_result;
+        
+        errorIsSmall = Math.abs(m_targetInputValue - m_currentInputValue) < errorTolerance;
```

Comment:
If the tolerance checks aren't enabled, set `errorIsSmall` (and its derivative counterpart) to be `false`.